src/index.ts

```diff
@@ -56,13 +60,13 @@ async function nitroModule(nitro: Nitro) {
   // Make sure runtime is transpiled
   nitro.options.externals.inline = nitro.options.externals.inline || [];
   nitro.options.externals.inline.push(
-    fileURLToPath(new URL("runtime/", import.meta.url)),
+    slash(fileURLToPath(new URL("runtime/", import.meta.url))),
```

Comment:
We might use `fileURLToPath` from `mlly` 1.6 which accepts URL and normalizes slash now ;)

Reply:
> We might use `fileURLToPath` from `mlly` 1.6 which accepts URL and normalizes slash now ;)

Nice, I'll try it now.

Reply:
done